airflow/dags/ingestion/ask-astro-load-astronomer-docs.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import os
+
+from include.tasks import split
+from include.tasks.extract.astro_docs import extract_astro_docs
+from include.tasks.extract.utils.weaviate.ask_astro_weaviate_hook import AskAstroWeaviateHook
```

Comment:
we should avoid using `AskAstroWeaviateHook` not since we have RC now
https://pypi.org/project/apache-airflow-providers-weaviate/1.3.0rc1/

Reply:
That's a separate task and it will be done as part of https://github.com/astronomer/ask-astro/issues/136